providers/pom.xml

```diff
@@ -25,9 +26,10 @@
   <artifactId>providers</artifactId>
   <packaging>pom</packaging>
   <modules>
-      <module>provider-jaxrs</module>
-      <module>provider-pojo</module>
-      <module>provider-rest-common</module>
-      <module>provider-springmvc</module>
+    <module>provider-jaxrs</module>
+    <module>provider-pojo</module>
+    <module>provider-rest-common</module>
+    <module>provider-springmvc</module>
+    <module>provider-metrics</module>
```

Comment:
provider means develop mode.
metrics is ......?

Reply:
Fixed，removed this module and inject service and contract via Schema Loader & Schema Factory